packages/backend-modules/discussions/graphql/schema.js

```diff
@@ -93,7 +93,15 @@ type mutations {
     targets: [CommentFeaturedTarget!]
   ): Comment!
 
-  suspendUser(id: ID!, until: DateTime, reason: String): User!
+  #
+  suspendUser(
+    id: ID!
+    #deprecated use numberDays instead
+    until: DateTime
```

Comment:
If deprecated, it should still be usable but mutation seems not to support that anymore.